x-pack/plugins/observability_solution/apm/public/components/alerting/ui_components/chart_preview/index.tsx

```diff
@@ -121,6 +122,7 @@ export function ChartPreview({
           legendPosition={'bottom'}
           legendSize={legendSize}
           locale={i18n.getLocale()}
+          theme={theme.colorMode === 'DARK' ? DARK_THEME : LIGHT_THEME}
```

Comment:
Could we consider using `COLOR_MODES_STANDARD.dark` from `@elastic/eui/src/services/theme/types` wherever it's currently being compared against `theme.colorMode`? I think it's used in like 4 or 5 places in the PR

It seems like a safer option in case there are any future changes.